### ISSUE TITLE
Sidebar Bug

### ISSUE THREAD

**Issue by Snorrrlax** [MystK/delta-bot-three#95]
Please see this post: https://www.reddit.com/r/ideasforcmv/comments/5i1ij9/whats_going_on_with_the_sidebar/

**Comment by hacksoncode** [comment]
This happened once before... I think about a month ago, and I hand-edited the sidebar to remove the extra amp;'s (unfortunately messing up the CSS). 

That time, it was visible even if the subreddit style (CSS) was in use. 

We also had a similar bug with exploding amp;'s in the users' wiki pages in the last version of DeltaBot. Maybe /u/PixelOrange could shed some light on it (sadly, we weren't really using a bug database back then, I don't think) if it proves really hard to find.

**Comment by wyantb** [comment]
Interesting!  Also bad, but interesting.

Note: this is displayed immediately below the section ```How to award a delta:
Include ∆ in your response to the comment(s) that changed your view, which can be achieved with one of the following: <JUNK HERE> Method | Works With```, and is displaying specifically as ```&amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;amp;nbsp;```

I'm guessing the following would fix?  Completely untested since the codepath in question isn't directly reachable at the moment.

```javascript
diff --git a/src/delta-boards-three.js b/src/delta-boards-three.js
index f5d92cb..b9e25a0 100644
--- a/src/delta-boards-three.js
+++ b/src/delta-boards-three.js
@@ -269,6 +269,7 @@ ${mapDataToTable(newHiddenParams.monthly)}
           .replace(/&gt;/g, '>')
           .replace(/amp;#/g, '#')
           .replace(/amp;\\/g, '\\')
+          .replace(/&(amp;)+nbsp;/g, '') // #95
         const currentAboutData = _.get(getAboutResponse, 'data')
 
         // start params
```

There at least seems to be some kind of precedent for this fn prettifying some of the output, anyway.

**Comment by Snorrrlax** [comment]
I think it might have something to do with this snippet of CSS we added recently to style the "how to award deltas" table:

    .side .md h3 { font-size: 0; height: 0; }
    .side .md h3 + table {
    position: relative;
    margin: 0 9px 0 51px;
    background: #fff; }
    .side .md h3 + table:before {
    position: absolute;
    top: 0;
    bottom: 0;
    left: -42px;
    content: "";
    width: 42px;
    border-right: 1px solid rgba(0,0,0,0.1);
    background: #f5f5f5; }

    .side .md h3 + table th, .side .md h3 + table td { border: 0; }
    .side .md h3 + table th { background: #f5f5f5; text-decoration: underline; text-align: left; border-bottom: 1px solid #eee; }
    .side .md h3 + table th:nth-of-type(1) {border-right: 1px solid #eee; }
    .side .md h3 + table tr {border-bottom: 1px solid #eee; }
    .side .md h3 + table td:nth-of-type(1) {border-right: 1px solid #eee; }
    .side .md h3 + table tr:nth-of-type(odd) td:nth-of-type(even) {background: #fcfcfc; }
    .side .md h3 + table tr:nth-of-type(even) td:nth-of-type(odd) {background: #fcfcfc; }

This issue has been made aware to us twice since this was added, but never before, and it occurs exactly where the "h3" is in the sidebar.

**Comment by MystK** [comment]
Please monitor if this is still happening

**Comment by roastchicken** [comment]
When not using the subreddit style, `&amp;amp;amp;amp;` displays under the "How to award a delta" section.

**Comment by MystK** [comment]
Fixed in latest commit, v3.5.1, but I don't have access to production server ATM. Will release in 2017

**Comment by MystK** [comment]
Fixed I think, please reopen if this happens again! From my quick testing it seems to have been fixed.
